src/main/resources/db/migration/0250/V259__Modules.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE accelerator.modules ADD COLUMN overview TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN preparation_materials TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN live_session_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN workshop_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN one_on_one_session_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN additional_resources TEXT;
+
+ALTER TABLE accelerator.modules ADD COLUMN phase_id INTEGER REFERENCES accelerator.cohort_phases;
+UPDATE accelerator.modules SET phase_id = 1 WHERE phase_id IS NULL;
+ALTER TABLE accelerator.modules ALTER COLUMN phase_id SET NOT NULL;
+
+CREATE TABLE accelerator.event_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL,
+    UNIQUE (name)
+);
+
+CREATE TABLE accelerator.events (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    module_id  BIGINT NOT NULL REFERENCES accelerator.modules ON DELETE CASCADE,
+    event_type_id INTEGER NOT NULL REFERENCES accelerator.event_types,
+    meeting_url TEXT,
+    slides_url TEXT,
+    recording_url TEXT,
+    start_time TIMESTAMP WITH TIME ZONE,
+    end_time TIMESTAMP WITH TIME ZONE,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+
+    UNIQUE(id, module_id, event_type_id)
+);
+
+CREATE TABLE accelerator.event_projects (
+    event_id BIGINT NOT NULL REFERENCES accelerator.events ON DELETE CASCADE,
+    project_id BIGINT REFERENCES projects ON DELETE CASCADE,
+    PRIMARY KEY (event_id, project_id)
+);
```

Comment:
A trailing newline would be good just so editors/formatters don't add one later on and cause the file to show up as edited.